src/init.cpp

```diff
@@ -69,6 +70,7 @@ static const bool DEFAULT_REST_ENABLE = false;
 static const bool DEFAULT_DISABLE_SAFEMODE = false;
 static const bool DEFAULT_STOPAFTERBLOCKIMPORT = false;
 
+std::shared_ptr<CConnman> g_connman;
```

Comment:
This was originally passed around differently, but refactored to essentially be a unique_ptr. So yes :)

Though it's worth mentioning that the global is only temporary until the rpc/qt interfaces learn how to accept/maintain an instance. So it's probably even more important to not encourage any sharing semantics.

Will change.